### PR TITLE
[MDB Ignore] Solves Merge Conflicts with #963, Solves Remaining Issues with Integration Tests

### PR DESCRIPTION
**Pull request by WrongEnd**
## About The Pull Request
Solves merge conflicts with #963 as per request of @out-of-phaze 

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: _maps/map_files/Pahrump-Sunset/RockSprings.dmm
Comment (Superlagg) [inline _maps/map_files/Pahrump-Sunset/RockSprings.dmm:45942]: might want to put the " before the ;

**Review by Superlagg — CHANGES_REQUESTED** [review]
I'm also going to wait for Phaze's approval, they know a lot more about this stuff than I do.

**Review by out-of-phaze — APPROVED** [review]
Well, it passes CI now, so it's an improvement no matter what. Ignore the mapdiffbot thing, that's because it's comparing it to the map prior to the fixes, which makes it die before it finishes. It's not actually still running, it's just a timestamp.

**Review by Superlagg — APPROVED** [review]

**Pull request merged**
